packages/plugin-core/src/_extension.ts

```diff
@@ -632,6 +632,14 @@ export async function _activate(
         MarkdownUtils.hasLegacyPreview()
       );
 
+      //used for enablement of export pod v2 command
+      if (!_.isUndefined(dendronConfig.dev?.enableExportPodV2)) {
+        VSCodeUtils.setContext(
+          DendronContext.ENABLEEXPORTPODV2,
+          dendronConfig.dev?.enableExportPodV2 ?? false
```

Comment:
isn't this equivalent to just `dendronConfig.dev?.enableExportPodV2`?
or are you guarding against the possibility that the value there would not be a boolean?

Reply:
5f2a11b